### ISSUE TITLE
Getting error when trying to install carbites

### ISSUE THREAD

**Issue by mehulagg** [nftstorage/carbites#3]
```
codespace ➜ .../packages/client/examples/node.js (main ✗) $ npm install carbites
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uglify-es@3.3.9: support for ECMAScript is superseded by `uglify-js` as of v3.13.0
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /workspaces/nft.storage/packages/client/examples/node.js/node_modules/carbites/bin/carbites.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/workspaces/nft.storage/packages/client/examples/node.js/node_modules/carbites/bin/carbites.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/codespace/.npm/_logs/2021-06-23T18_14_58_379Z-debug.log
```

[2021-06-23T18_14_58_379Z-debug.log](https://github.com/nftstorage/carbites/files/6704096/2021-06-23T18_14_58_379Z-debug.log)

**Comment by mehulagg** [comment]
This problem happens with 1.0.3 version.
1.0.2 version is working.

**Comment by alanshaw** [comment]
Thanks for reporting I'll look into it asap!

**Comment by alanshaw** [comment]
@mehulagg sorry about that - resolved in v1.0.4
